package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "eslint-config-digitalbazaar",
+  "version": "1.0.0",
```

The comments about package.json are specific to how we make releases using some scripts...  so...
```suggestion
  "version": "0.0.1-0",
```